### PR TITLE
Edit master

### PR DESCRIPTION
**Pull request by NNieuwenhuis**

### CODE REVIEW

**Review by lukeelten — APPROVED** [review]

File: src/app/components/module-component/edit/module-edit.component.ts [hunk 1]
```diff
@@ -97,6 +97,7 @@ export class ModuleEditComponent extends RestrictedComponent implements OnInit {
   }
   selectTeachingMaterialType(type: string): void {
     this.selectedTeachingMaterialType = type;
+    console.log(type);
```
Comment (Lehren) [inline src/app/components/module-component/edit/module-edit.component.ts]: Console log

File: src/app/components/module-component/edit/module-edit.component.ts [hunk 2]
```diff
@@ -107,6 +108,7 @@ export class ModuleEditComponent extends RestrictedComponent implements OnInit {
         }
       });
       if (found) {
+        console.log(this.selectedTeachingMaterialType);
```
Comment (Lehren) [inline src/app/components/module-component/edit/module-edit.component.ts]: Console log

File: src/app/components/module-component/edit/module-edit.component.html [hunk 3]
```diff
@@ -29,7 +29,7 @@
     </div>
     <div class="col-2">
       <input onkeyup="if(this.value.length > 2 || this.value > 40) this.value = 40;" mdbActive id="practical-hours" class="form-control-sm" type="number" min="0" max="50" ondrop="return false;" (paste)="false" onkeypress="return event.charCode >= 48 && event.charCode <= 57" [(ngModel)]="output.practical_hours_week">
-      <label [ngClass]="{'active' : output.practical_hours_week}" for="practical-hours">* Practical hours / week</label>
+      <label [ngClass]="{'active' : output.practical_hours_week || output.practical_hours_week === 0}" for="practical-hours">* Practical hours / week</label>
```
Comment (Lehren) [inline src/app/components/module-component/edit/module-edit.component.html]: You need to do this for the rest too. Credits and lectures per week

File: src/app/components/module-component/edit/module-edit.component.ts [hunk 4]
```diff
@@ -116,8 +116,13 @@ export class ModuleEditComponent extends RestrictedComponent implements OnInit {
   // send save data
   save(): void {
     this.input = new EditableModuleInput(this.output);
-    this.backendService.updateEditableModule(this.output.id, this.input);
-    this.location.back();
+    this.backendService.updateEditableModule(this.output.id, this.input).subscribe(() => {
+      this.location.back();
+    }, (error) => {
+      console.log("Error");
+      console.log(error);
```
Comment (Lehren) [inline src/app/components/module-component/edit/module-edit.component.ts:125]: Double Console log

**Pull request merged**